portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,54 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+  background-color: #fbfcfc;
+}
+
+
+/* Styling the header */
+
+.header {
+  overflow: hidden;
+  background-color: #fbfcfc;
+  padding: 20px 10px;
+  margin-left: auto;
+  margin-right:auto;
+  width: 750px;
+}
+
+/* Stylize image in header as a profile photo */
+
+.header img {
+  float: left;
+  border-radius: 70%;
+  border: 2px solid black;
+  height: 150px;
+  width: 150px;
+  margin-right: 20px;
+}
+
+/* Interactive header links */
+
+.header a {
+  margin-left: 5px;
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 12px;
+  text-decoration: none;
+  font-size: 18px;
```

Comment:
nit: font sizes should normally be specified in "em" units, which will scale with the user's font size setting in their browser

Reply:
That makes sense, I'll make the change.